lib/ui/pages/main_page.dart

```diff
@@ -5,33 +5,101 @@ class MainPage extends StatelessWidget {
+                        icon: Icons.person_outline,
+                        iconColor: AppColor.greyColor,
+                        shadowColor: AppColor.greyColor.withOpacity(0.6),
+                        radius: 21,
+                        onPressed: () {
+                          //TODO:Navigate to user profile screen
+                        },
+                      ),
+                      Column(
+                        children: const [
+                          Text(
+                            'Delivering to',
+                            style: TextStyle(
+                                color: AppColor.blackColor, fontSize: 15),
+                          ),
+                          Text(
+                            "Leonard",
+                            style: TextStyle(
+                                color: AppColor.redColor, fontSize: 18),
+                          )
+                        ],
+                      ),
+                      Row(
+                        children: [
+                          CircularIconButton(
+                            icon: Icons.search,
+                            iconColor: AppColor.greyColor,
+                            shadowColor: AppColor.greyColor.withOpacity(0.6),
+                            radius: 21,
+                            onPressed: () {
+                              //TODO:Implete search functionality
+                            },
+                          ),
+                          CircularIconButton(
+                            icon: Icons.shopping_cart_outlined,
+                            iconColor: AppColor.greyColor,
+                            shadowColor: AppColor.greyColor.withOpacity(0.6),
+                            radius: 21,
+                            onPressed: () {
+                              //TODO:Navigate to cart screen
+                            },
+                          )
+                        ],
+                      ),
+                    ],
+                  ),
                 ),
-                onTap: () {
-                  Navigator.pushNamed(
-                    context,
-                    AppRoutes.examplePage,
-                    arguments: "some value",
-                  );
-                })
-          ],
+                SingleChildScrollView(
+                  child: Padding(
+                    padding: const EdgeInsets.all(10.0),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.stretch,
+                      mainAxisSize: MainAxisSize.min,
+                      children: [
+                        Text("Let's Eat !!!",style: AppTypography.blackw400.copyWith(fontSize: 15),),
+                        const RestaurantOfTheWeekCard(),
+                        Text("What are you craving for ?",style: AppTypography.blackw400.copyWith(fontSize: 15),),
+                        DishSuggestionCard(),
```

Comment:
Prefer const with constant constructors
```suggestion
                        const DishSuggestionCard(),
```